.claude/commands/commit.md

```
@@ -0,0 +1,110 @@
+# Claude Command: Commit
+
+This command helps you create well-formatted commits with conventional commit messages.
+
+## Usage
+
+To create a commit, just type:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)